elasticsearch/src/test/java/com/tcdi/zombodb/test/ZomboDBTestCase.java

```diff
@@ -31,6 +31,7 @@
 import org.elasticsearch.node.Node;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.skyscreamer.jsonassert.JSONAssert;
```

Comment:
I thought it may have better output on failing (for debugging). Will change it back